src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Component {
 }
```

Comment:
You need to add `@Target` as well (check common mistakes)

Reply:
agree